src/ro/catalog/servicii/ServiceCatalog.java

```diff
@@ -15,16 +18,16 @@ public static Catalog creazaCatalog(int grupa){
         return new Catalog(grupa);
     }
 
-    static public Materie creazaMaterie(String denumire, int nrCredite){
+    static public Materie creareMaterie(String denumire, int nrCredite){
```

Comment:
I don't see any audit service being called on these methods from etapa1, you only call it (and write to data.csv) for ServiceUtilizatori, right?

Reply:
Yes, that's right, I will solve it.